### PR TITLE
CHG0020161 MCS SILVER - Enable Audit Log and MTU Probing

### PR DESCRIPTION
**Pull request by sbarre-esit**

### CODE REVIEW

**Comment by sbarre-esit** [comment]
Git got confused about some of the deletes and new files. Look at f3d6259 for the new notice separated out.

**Review by wmhutchison — APPROVED** [review]
👍

**Comment by wmhutchison** [comment]
> 👍

@sbarre-esit Acknowledged regarding the specific commit.

File: notifications/11-17-20-1500.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
 
 **What is happening?**
 
-CHG0019473 MCS - Drain pods from ociopf-p-211 and troubleshoot SAN connection
+CHG0020161 MCS SILVER - Enable Audit Log and MTU Probing
 
-Drain pods from the node to ensure no disruption to the applications.
+OpenShift 4 generates audit logs of the hosts and API but does not collect them into the aggregated logging system on the platform by default. This change will adjust the logging configuration to ensure the logs are captured and retained.
```
Comment (mitovskaol) [inline notifications/11-17-20-1500.md:6]:
```suggestion
OpenShift 4 generates audit logs of the hosts and API , however,  aggregated logging is not enabled on the platform by default. This change will adjust the logging configuration to ensure the logs are captured, aggregated and retained in the Silver cluster.
```

**Review by mitovskaol — APPROVED** [review]

**Pull request merged**